test/controllers/advanced_search_controller_test.rb

```diff
@@ -0,0 +1,81 @@
+require 'test_helper'
+
+class AdvancedSearchControllerTest < ActionDispatch::IntegrationTest
+  test 'advanced search route exists and renders' do
+    get '/advanced-search'
+    assert_response :success
+  end
+
+  test 'initial form state' do
```

Comment:
It's worth noting that the criteria here will likely change as we fiddle with the advanced search form, but we can deal with that when we come to it.